install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Install ocaml/opam
+if [ -f "/etc/arch-release" ]; then #detect arch-linux
+        pacman -S opam
+elif [[ "$OSTYPE" == "linux-gnu"* ]]; then
+        apt install opam
+elif [[ "$OSTYPE" == "darwin"* ]]; then #MacOS
+        #Check if homebrew is installed
+        which=$(command -v which -s brew)
+        if [[ $which != 0 ]] ; then
+            # Install Homebrew
+            ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+        else
+            brew update
+        fi
+        brew install gpatch
+        brew install opam
+elif [[ "$OSTYPE" == "freebsd"* ]]; then
+        cd /usr/ports/devel/ocaml-opam || exit
+        make install
+elif [[ "$OSTYPE" == "openbsd"* ]]; then
+        pkg_add opam
+else
+    "Unknown OS type, please install opam/ocaml manually."
+fi
+
+# Initailize a switch
+opam init
+eval "$(opam env)"
```

Comment:
This is for the case that the user didn't want to update their bash profile in `opam init`, right? If that happens, how do we solve the problem that dune might not be found when running the `ofronds` command? Should we maybe also run `eval "$(opam env)"` every time before internally running a dune command?

Reply:
Hmm, that will be kind of annoying though, (to run eval everytime before running `dune` command). Do you know if we can enforce updating bash profile in opam init somehow?

Reply:
I think we could do that via `opam init --auto-setup`. We should discuss if that would be too intrusive. I think it should be fine.